server/src/utils/general-controller.ts

```diff
@@ -10,30 +10,53 @@ class GeneralController {
         this.name = name;
     }
 
+    /**
+     * Creates new item in model
+     * @param params {object}
+     * @param params.data {object} data to store
+     * @returns {Promise<object>} created item
+     */
     async create(params: any) {
```

Comment:
I see that in all cases params are type of `any`, on client side it seems that contract is already defined. We can here by more specific what type of parameters we can really expect. Only hint we got are the jsdocs

Reply:
I added {data: any} schema